js/map.js

```diff
@@ -54,13 +59,42 @@ function generateMonsters(){
     monsters = [];
     let numMonsters = level+1;
     for(let i=0;i<numMonsters;i++){
-        spawnMonster();
+        spawnMonster(level);
```

Comment:
How about changing that for something like that:
```js
function generateMonsters(level){
  const numMonsters = level+1;
  const monsters = numMonters.map(() =>
    spawnMonster(level)
  )
}

Reply:
The owner of the project, wasn't using map... but i think i will give it a try. Thanks for the review. :D